### PR TITLE
Dpquigl/san

### PR DESCRIPTION
**Pull request by dpquigl**

### CODE REVIEW

File: utils/certs/ca.cnf [hunk 1]
```diff
@@ -32,6 +32,7 @@ basicConstraints = critical,CA:true,pathlen:1
 [ signing_policy ]
 organizationName	= supplied
 commonName		= supplied
+subjectAltName		= supplied
```
Comment (mjmac) [inline utils/certs/ca.cnf:35]: Whitespace issue?

File: src/control/security/config.go [hunk 2]
```diff
@@ -61,7 +60,6 @@ func (tc *TransportConfig) String() string {
 // component. ServerName is only needed if the config is being used as a
 // transport credential for a gRPC tls client.
 type CertificateConfig struct {
-	ServerName      string           `yaml:"server_name,omitempty"`
```
Comment (mjmac) [inline src/control/security/config.go:64]: The example configs and documentation will probably need to be updated as part of this work.

**Review by mjmac — COMMENTED** [review]
Looks good, hopefully it works OK. From a security perspective, this is basically status quo with the previous implementation, right? We're effectively saying that we don't really care about the peer's hostname or domain name, and that's the same as before, if I understand correctly.

File: src/control/security/config.go (hunk 2 above)
Comment (dpquigl) [reply src/control/security/config.go:64]: I dont believe any of the sample configs have that value in them anymore. I will check to make sure though.

**Pull request closed without merging**